packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/__tests__/PdfViewer.test.tsx

```diff
@@ -4,9 +4,12 @@ import { render, waitFor, screen } from '@testing-library/react';
 import 'web-streams-polyfill/es2018';
 import PdfViewer from '../PdfViewer';
 import { document as doc } from 'components/DocumentPreview/__fixtures__/Art Effects.pdf';
+import { setupBeforeRenderPdf } from 'setupTestsUtil';
 
 describe('PdfViewer', () => {
   it('renders PDF document', async () => {
+    // By experiment, it seems that setupBeforeRenderPdf is needed
+    setupBeforeRenderPdf();
```

Comment:
By experiment, it seems that setupBeforeRenderPdf is needed.

I welcome suggestions if I missed a place to call `removeDOMRect()`

Reply:
Why? If it's removed at the end of previous tests, then it shouldn't be necessary here.

Reply:
More experimenting, found it is not needed.